tchannel-benchmark/src/main/java/com/uber/tchannel/benchmarks/PingPongServerBenchmark.java

```diff
@@ -83,9 +89,6 @@ public void benchmark() throws Exception {
                 request,
                 Pong.class
         );
-
-        f.get();
```

Comment:
I believe this is related to serial handling of requests in the RequestDispatcher…I'll file a bug and see if I can put in a quick fix w/ an executor / ThreadPool